util/print/printWinner.js

```diff
@@ -0,0 +1,14 @@
+import { Console } from "@woowacourse/mission-utils";
+
+import findMoveLonger from "../find/findMoveLonger.js";
+
+export default function printWinner(winners, carMoveObj) {
+    if (winners.length === 0) {
+        Console.print('우승자가 가려지지 않았습니다.');
+
+        findMoveLonger(carMoveObj);
+
+    } else {
+        Console.print(`최종 우승자 : ${winners.join(', ')}`);
```

Comment:
```suggestion
    if (winners.length === 0) {
        Console.print('우승자가 가려지지 않았습니다.');
        return findMoveLonger(carMoveObj);
    } 
     if (winners.length !== 0) {{
       return  Console.print(`최종 우승자 : ${winners.join(', ')}`);
       }
```

이렇게 작성하는게 가독성이 조금 더 좋을 것 같습니다!